### PR TITLE
blog: PELUX 2.0 release announcement

### PR DESCRIPTION
**Pull request by sashko**
Signed-off-by: Oleksandr Kravchuk <oleksandr.kravchuk@pelagicore.com>

### CODE REVIEW

File: _posts/2018-06-25-PELUX-2.0.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+We are happy to announce the release of PELUX 2.0!
+
+This major release contains the following changes:
+* Adaptation to Yocto "Rocko" 2.4.
+* [Automotive Reference Platform](https://www.youtube.com/watch?v=XVF19cC98Xs) support
+* SOTA updates using [swupdate](https://github.com/sbabic/swupdate) with the help of [meta-swupdate](https://github.com/sbabic/meta-swupdate)
+* ACPI support for Intel NUC
+* Improved documentation
+* Improved CI
```
[the remark this answers is not in the document]
Comment (sashko) [reply _posts/2018-06-25-PELUX-2.0.md]: Sure, just write down the sentence and I'll add it.

[the remark this answers is not in the document]
Comment (thenor) [reply _posts/2018-06-25-PELUX-2.0.md]: I think it should be something like
"* Improved CI
    - including running poky tests for qemu"
Something like that?

**Comment by AshleyNair** [comment]
Made a few changes to the text

Today, we are happy to announce the release of PELUX 2.0!  We would like to thank all for your tremendous support and feedback. This release comes with many improvements and new features as well. 

The highlights of this major release are:
1)	Adaptation to Yocto "Rocko" 2.4.
2)	Integration with Qt 5.10
3)	Automotive Reference Platform [ARP] support: This is a hardware platform that has been developed in house for automotive and hence the name ARP. More details about the ARP board can be found at (https://www.youtube.com/watch?v=XVF19cC98Xs) 
4)	SOTA updates using [swupdate](https://github.com/sbabic/swupdate) with the help of [meta-swupdate](https://github.com/sbabic/meta-swupdate)
5)	ACPI support for Intel NUC
6)	Improved documentation

**Comment by thenor** [comment]
@sashko I believe list item 7 is removed in @AshleyNair's comment.

**Pull request merged**